build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -127,31 +161,105 @@ export abstract class PackageCommand<
 		}
 	}
 
+	/**
+	 * Returns an array of the release group packages that should be processed.
+	 */
+	private getReleaseGroupPackages(
+		ctx: Context,
+		releaseGroup: ReleaseGroup,
+		rootPackageOnly: boolean,
+	): PackageDetails[] {
+		this.info(`Finding packages for release group: ${releaseGroup}`);
+		if (rootPackageOnly) {
+			const rg = ctx.repo.releaseGroups.get(releaseGroup);
+			assert(rg !== undefined);
+			return [{ directory: rg.repoPath, kind: "releaseGroupRootPackage" }];
+		}
+
+		return ctx.packagesInReleaseGroup(releaseGroup).map((p) => {
```

Comment:
Again, for another PR, but I think a decent amount of logic over in monoRepo.ts could be replaced by a call to [releaseGroups](https://www.npmjs.com/package/find-packages) to find these packages.